### ISSUE TITLE
Expose play/pause button image in View

### ISSUE THREAD

**Issue by rudolf-adamkovic** [enfluid/apple-audio-bar#62]
This needs to be fixed:
```swift
var playPauseButtonImage: UIImage {
    switch view.playPauseButtonEvent {
        case .userDidTapPlayButton: return loadImage(withName: "Play Button")
        case .userDidTapPauseButton: return loadImage(withName: "Pause Button")
    }
}
```
I.e. `playPauseButtonEvent` is a different thing. Instead, we want to expose `playPauseButtonImage`.

**Comment by nasakinmaxim** [comment]
@salutis
>we want to expose `playPauseButtonImage`.

The functional core doesn't need to know about the world (in our case, assets).
What if we want to have a different image with a different name? For that, we'd need to change the functional core.

**Comment by rudolf-adamkovic** [comment]
@nasakinmaxim 

> The functional core doesn't need to know about the world (in our case, assets).

That's right. It should be an `enum`, e.g. `.play` and `.pause` images. The imperative shell will then map them to actual assets.

**Comment by nasakinmaxim** [comment]
Irrelevant due to #74. Closing.

**Comment by rudolf-adamkovic** [comment]
Closed #74, reopening this one.
